### PR TITLE
Add switch_tcpdump tool documentation

### PR DESCRIPTION
**Pull request by rr-santos**
Add documentation for the new switch_tcpdump tool, including how it works and usage examples.

Signed-off-by: Ricardo Santos <ricardo.santos@bisdn.de>

### CODE REVIEW

File: tools/ofdpa_client_tools.md [hunk 1]
```diff
@@ -130,6 +130,40 @@ sudo journalctl -fu ofdpa
 Apr 14 12:37:59 agema-ag7648 ofdpa[10452]: ofdbFlowPolicyAclEntryValidate: Invalid ethertype for IPv6 match fields.
 ```
 
+## Traffic capture with tcpdump
+
+The `switch_tcpdump` tool allows users to capture traffic on the switch port network interfaces created by `baseboxd` seen in BISDN Linux, e.g. `port1`.
```
Comment (KanjiMonster) [inline tools/ofdpa_client_tools.md:135]:
```suggestion
The `switch_tcpdump` tool allows users to capture ingress traffic on the switch port network interfaces created by `baseboxd` seen in BISDN Linux, e.g. `port1`.
```

File: tools/ofdpa_client_tools.md [hunk 2]
```diff
@@ -130,6 +130,40 @@ sudo journalctl -fu ofdpa
...
+Internally, this is done by adding an ACL table entry matching on the desired ingress port with the [ACL table entry management tool](#acl-table-entry-management) and sending its traffic to controller, capturing traffic with `tcpdump`, and then deleting the created entry from the ACL table.
+
+Since captured traffic is sent to controller, matched packets are not routed directly through the ASIC, therefore we do not recommend doing performance measurements with this tool.
```
Comment (KanjiMonster) [inline tools/ofdpa_client_tools.md:138]: The traffic is still routed through the ASIC, so there should be no performance impact to the flows being matched. The ACL flow matched packets are copied to the controller, not redirected (and then forwarded).

**Review by KanjiMonster — CHANGES_REQUESTED** [review]
I think it needs to mention that it can only capture ingress traffic, and not both directions.

File: tools/ofdpa_client_tools.md (hunk 2 above)
Comment (rr-santos) [reply tools/ofdpa_client_tools.md:138]: When creating flows tool that are sent to controller with the ACL CLI tool, the clearAction flag is added. This prevents duplicate packets (forwarded through the ASIC and also by the controller after receiving a copy)

Comment (KanjiMonster) [reply tools/ofdpa_client_tools.md:138]: Ah, you are right. Somehow missed that.

**Pull request merged**